catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/UdpStreamMonitor.java

```diff
@@ -363,7 +364,17 @@ public void destroy(int arg) {
 
     private void shutdown() {
         if (eventLoopGroup != null) {
-            eventLoopGroup.shutdownGracefully();
+            try {
+                eventLoopGroup.shutdownGracefully()
+                        .sync();
+                if (channelFuture != null) {
+                    channelFuture.channel()
+                            .closeFuture()
+                            .sync();
+                }
+            } catch (InterruptedException e) {
```

Comment:
If the first sync throws an `InterruptedException` what state does that leave the `channelFuture`? We need to ensure both on closed/shutdowned and synced.